docs/reducing-contract-size/examples.md

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_position: 1
+---
+
+# Advice & examples
+
+This page is made for developers familiar with lower-level concepts who wish to reduce their contract size significantly, perhaps at the expense of code readability.
+
+Some common scenarios where this approach may be helpful:
+
+- contracts intended to be tied to one's account management
+- contracts deployed using a factory
+- future advancements similar to the EVM on NEAR
+
+There have been a few items that may add unwanted bytes to a contract's size when compiled. Some of these may be more easily swapped for other approaches while others require more internal knowledge about system calls.
+
+## Small wins
+
+1. See the section in best practices on [creating smaller binaries](/best-practices#compile-smaller-binaries) using `opt-level` in the Cargo manifest.
+2. Ensure that your manifest doesn't contain `rlib` unless it needs to. Some NEAR examples have included this:
+
+  :::caution Possibly adds unnecessary bloat
+
+  ```toml
+  [lib]
+  crate-type = ["cdylib", "rlib"]
+  ```
+  :::
+
+  when it could be:
+
+  :::tip
+
+  ```toml
+  [lib]
+  crate-type = ["cdylib"]
+  ```
+  :::
+
+3. Avoid using the [`serde` crate](https://crates.io/crates/serde) and perhaps experiment with the [miniserde crate](https://crates.io/crates/miniserde) or the [rjson crate](https://crates.io/crates/rjson).
+4. When using assertions or guards, avoid using the standard `assert` macros like [`assert!`](https://doc.rust-lang.org/std/macro.assert.html), [`assert_eq!`](https://doc.rust-lang.org/std/macro.assert_eq.html), or [`assert_ne!`](https://doc.rust-lang.org/std/macro.assert_ne.html) as these may add bloat for information regarding the line number of the error.
```

Comment:
Might want to include the usage of `unwrap` or `expect` also adding similar bloat because it encodes the same debug information. Also rust's `panic!()` macro

Reply:
thanks, added commit for this:
https://github.com/near/sdk-docs/pull/37/commits/a786224299eb0788e12f0bbf9c3e567f51c8454f